PKGBUILD

```
@@ -0,0 +1,33 @@
+pkgname=napture
+pkgver=1.2.2
+pkgrel=1
+pkgdesc="Napture is a simple browser using a modified version of the http protocol called buss, it uses Lua instead of JavaScript and has support for most HTML and CSS tags, it uses a custom DNS found at api.buss.lol."
```

Please use the same description as in the MetaInfo
```suggestion
pkgdesc="An alternative for the World Wide Web"
```